src/app/components/updatedList/search.model.js

```diff
@@ -0,0 +1,12 @@
+(function() {
```

Comment:
There is an exact copy of this one in advancedSearch. Better to take that one and move it into the shared module. Make sure that you fix the import for advancedSearch after you move it.